src/main/resources/templates/telegram/start.ftl

```diff
@@ -1,3 +1,4 @@
 <b>Вітаємо, Shpalta IF бот повідомлятиме Вас про нові пости на ${appBasePath}</b>
 
-Категорії, сповіщення з яких Ви отримуватимете, можна налаштувати, натиснувши на кнопку <b>"Налаштувати сповіщення"</b> або відправивши /settings
\ No newline at end of file
+Категорії, сповіщення з яких Ви отримуватимете, можна налаштувати, натиснувши на кнопку <b>"Налаштувати
```

Comment:
Why changed formatting?

Reply:
Sorry, it has been changed by autoformat